src/components/P5AP_ChallengePage/ChallengePage.jsx

```diff
@@ -1,34 +1,103 @@
-import React, { useState } from "react";
-import { MOCK_CHALLENGES } from "../../mock-data/mockChallenges";
+import React, { useState, useEffect, useContext } from "react";
+import { readData } from "../../utils/Functions/functions";
 import ChallengeCard from "../ChallengeCard/ChallengeCard";
 import DashboardNavbar from "../DashboardNavbar/DashboardNavbar";
 import filterIcon from "../../assets/images/filterIcon.svg";
 import FilterBoard from "../P5AP_FilterBoard/FilterBoard";
+import { PageContext } from "../../contexts/PageContext";
 import "./ChallengePage.scss";
 
 const ChallengePage = () => {
+  const { userFeedback } = useContext(PageContext);
+  console.log("ChallengePage mounted, userFeedback:", userFeedback);
```

Comment:
I noticed there are quite a few console.log left from testing. These should be removed